lego/apps/users/fields.py

```diff
@@ -27,6 +27,19 @@ def __init__(self, field_kwargs=None, **kwargs):
         super().__init__(**kwargs)
 
 
+class PublicUserWithGroupsField(serializers.PrimaryKeyRelatedField):
+    def use_pk_only_optimization(self):
+        return False
+
+    def to_representation(self, value):
+        from lego.apps.users.serializers.users import (
+            PublicUserWithAbakusGroupsSerializer,
+        )
```

Comment:
Any reason not to import this at the top of the file?

Reply:
Just copy pasta from the other one, but I suspect it is because of circular deps of some kind, but not 100% sure tho

Reply:
Jup, checks out :ok_hand: